frontend/hub/api/request.ts

```diff
@@ -57,6 +59,41 @@ async function hubRequestCommon<T extends object | TaskResponse>(
   };
 }
 
+export async function hubPostRequestFile(
+  url: string,
+  file: Blob,
+  repository?: string,
+  signed_collection?: string,
+  signal?: AbortSignal
+): Promise<unknown> {
```

Comment:
`Promis<unknown>` .. given the tsc error I'm *guessing* for async functions, you'd just want unknown, not Promise<unknown> .. unless that blows up somewhere else :) 

Or return an actual value and use that for the type? Since we're waiting for a task, maybe returning the TaskResponse would make sense